### PR TITLE
add --buffersize option to rflash command

### PR DESCRIPTION
**Pull request by gurevichmark**
1. Add `--buffersize` option to `rflash` command.
2. Pass in verbose flag to first invocation of `check_bmc_status_with_ipmitool()` - code review comment from PR #2807

### CODE REVIEW

File: xCAT-server/lib/xcat/plugins/ipmi.pm [hunk 1]
```diff
@@ -1933,7 +1943,7 @@ sub do_firmware_update {
             "Running ipmitool command $cmd failed: $output");
     }
     $exit_with_success_func->($sessdata->{node}, $callback,
-        "Success to update firmware. FRU information will be populated in a few minutes.");
+        "Success updating firmware. FRU information will be populated in a few minutes.");
```
Comment (whowutwut) [inline xCAT-server/lib/xcat/plugins/ipmi.pm]: Thinking we might want to be a little more transparent as to what we are doing here... 

Line 1871, when we do `chassis power off`, suggest printing a message 
_Preparing to upgrade firmware, powering chassis off..._

Line 1940,  before doing `chassis power on`, we can print a message:
_Firmware updated, powering chassis on to populate FRU information..._ 

Then here in this line we can shorten to just: 
_Success updating firmware._

**Comment by whowutwut** [comment]
Helps with debugging issue #2706

**Comment by gurevichmark** [comment]
Added new commit to improve progress messages:

```
-bash-4.2$  sudo /opt/xcat/bin/rflash c460c133 /root/hpm/garrison/8335GTB_820.1642.20170327n_update.hpm
c460c133: rflash started, please wait.......
c460c133: Preparing to upgrade firmware, powering chassis off...
c460c133: rflashing ... See the detail progress : "tail -f /var/log/xcat/rflash/c460c133.log"
c460c133: Firmware updated, powering chassis on to populate FRU information...
c460c133: Success updating firmware.
-bash-4.2$
```

**Comment by whowutwut** [comment]
@zet809  FYI, I've cherry-picked this pull request to 2.13 branch directly

**Comment by zet809** [comment]
Thx, @whowutwut

**Pull request merged**